docs/RFCS/explain_debug.md

```diff
@@ -15,7 +15,8 @@ underlying key-value pairs that were accessed:
   underlying key-value store.
 * the third column will contain a pretty-printed version of the value.
 * the fourth column will contain a boolean indicating whether the row would
-  have been included in the normal output or not.
+  have been included in the normal output or not. This field is NULL for an key
+  that is not the last one on the row.
```

Comment:
Well the grammar sucks (`s/an key/any key/g`, `s/on the row/in the row/g`). Take a look at the example where the `Output` column is NULL except for the final key for each `RowIdx`.